packages/salesforcedx-vscode-soql/src/sfdx.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Connection } from '@salesforce/core';
+import { ChannelService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
+import { DescribeSObjectResult } from 'jsforce';
+import * as vscode from 'vscode';
+import { nls } from './messages';
+
+export const channelService = ChannelService.getInstance(
+  nls.localize('soql_channel_name')
+);
+
+const sfdxCoreExtension = vscode.extensions.getExtension(
```

Comment:
We shouldn't be using this pattern anymore, replace it with the modules on `salesforcedx-utils-vscode` e.g. https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

Reply:
Yes.  @jgellin-sf  has this covered on another PR on top of this one

Reply:
@lcampos FYI here is the PR for reference: https://github.com/forcedotcom/salesforcedx-vscode/pull/2837